### DISCUSSION TITLE
Add a feature or maybe completely changing it

### DISCUSSION THREAD

**Discussion by Hanzla-Double3** [importantimport/urara#80] [General]
I want to add a login feature and authentication (blah blah blah) . Want to made it like social media type of thing. But I don't have any knowledge about frontend and js (or ts). Anyways initially I was thinking of somehow extract and removing the backend logic and add an api instead. Cuz I already have a backend api of my project (of kinda same type) but never finished it due to frontend. If any one of you guys have any suggestions or can help me kindly reach out. Thank you in anticipation.

**Comment by kwaa** [comment]
> I want to add a login feature and authentication.

For Urara (SvelteKit), I recommend [Lucia Auth](https://lucia-auth.com/getting-started/sveltekit), but Urara doesn't take this usage scenario, so you'll need to modify it yourself.

> But I don't have any knowledge about frontend and js (or ts).

You may need to learn JS/TS and Svelte first.
